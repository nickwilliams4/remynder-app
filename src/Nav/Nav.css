.Nav a,
.Nav a:active,
.Nav a:focus,
.Nav a:visited {
  color: black;
}

.Nav a:hover {
  color: blue;
  text-decoration: solid;
}

.Nav a:not(:last-of-type) {
  margin-right: 120px;
}

.remynder {
  padding-bottom: 10px;
}

@media screen and (max-width: 400px) {
  .Header__not-logged-in, .Header__logged-in {
    position: absolute;
    margin-top: -28px;
    margin-left: 310px;
  }
}