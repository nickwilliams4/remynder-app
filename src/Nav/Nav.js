import { Link } from 'react-router-dom'
import React, { Component } from 'react'
import Content from '../Content/Content'
import './Nav.css'

export default class Nav extends Component {
  render() {
    return (
      <Content className='Nav'>
        <Link to='/'>
          Remynder
        </Link>
        <Link to='/ReturningUser'>
          Log In
        </Link>
      </Content>
    )
  }
}