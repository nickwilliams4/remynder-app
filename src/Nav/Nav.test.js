import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Nav from './Nav'

describe(`Note component`, () => {
  const props = {
    className: 'test-class-name',
  }

  it('renders the Nav', () => {
    const wrapper = shallow(<Nav {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})