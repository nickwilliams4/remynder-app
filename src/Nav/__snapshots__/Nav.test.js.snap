// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component renders the Nav 1`] = `
<nav
  className="Nav"
>
  <Link
    to="/"
  >
    Remynder
  </Link>
  <div
    className="Header__not-logged-in"
  >
    <Link
      to="/SignUpForm"
    >
      Register
    </Link>
    <Link
      to="/ReturningUser"
    >
      Log in
    </Link>
  </div>
</nav>
`;
